geoh5py/objects/object_base.py

```diff
@@ -220,7 +220,11 @@ def default_type_uid(cls) -> uuid.UUID:
         ...
 
     def copy_from_extent(
-        self, bounds: np.ndarray, parent=None, copy_children: bool = True
+        self,
+        bounds: np.ndarray,
+        parent=None,
+        copy_children: bool = True,
+        clear_cache: bool = False,
```

Comment:
**issue (doc):** obsolete doc of parameters

The 3 last parameters are not documented, while the doc mentions non existing parameter `attributes`